pkg/distributor/distributor.go

```diff
@@ -45,6 +45,21 @@ var (
 		Name:      "distributor_received_samples_total",
 		Help:      "The total number of received samples.",
 	}, []string{"user"})
+	incommingSamples = promauto.NewCounterVec(prometheus.CounterOpts{
+		Namespace: "cortex",
+		Name:      "distributor_samples_in_total",
+		Help:      "The total number of samples that have come in to the distributor, including rejected or deduped samples.",
+	}, []string{"user"})
+	nonHASamples = promauto.NewCounterVec(prometheus.CounterOpts{
```

Comment:
Unless I am misreading the behavior, this will only increment once per payload, so is really the total number of received requests that didn't contain both HA labels.